README.md

```diff
@@ -76,7 +76,11 @@ Handler to be called when the user taps the button
 ### 2. GoogleSignin
 
 ```js
-import { GoogleSignin, GoogleSigninButton, statusCodes } from 'react-native-google-signin';
+import {
```

Comment:
I believe that with `"printWidth": 100` I think this diff should not be here - none of the js diffs should be here, in fact..?